{% extends "layout.html" %}
{% block content %}

<h3 class="param">Dinucleotide</h3>
<div class="parameters-container m-5">
<div class="parameter">
    <h5  class="parameter-heading">1. Arule</h5>
    <p class="parameter-description">ProtParam is a tool that allows the computation of various physical and chemical parameters for a given protein stored in Swiss-Prot or TrEMBL or for a user-entered protein sequence. The computed parameters include the molecular weight, theoretical pI, amino acid composition, atomic composition, extinction coefficient, estimated half-life, instability index, aliphatic index, and grand average of hydropathicity (GRAVY). </p>
    </div>
    <div>
    <h5  class="parameter-heading">2. Trule</h5>
    <p class="parameter-description">ProtParam is a tool that allows the computation of various physical and chemical parameters for a given protein stored in Swiss-Prot or TrEMBL or for a user-entered protein sequence. The computed parameters include the molecular weight, theoretical pI, amino acid composition, atomic composition, extinction coefficient, estimated half-life, instability index, aliphatic index, and grand average of hydropathicity (GRAVY). </p>
    </div>
    <div>
    <h5  class="parameter-heading">3. Grule</h5>
    <p class="parameter-description">ProtParam is a tool that allows the computation of various physical and chemical parameters for a given protein stored in Swiss-Prot or TrEMBL or for a user-entered protein sequence. The computed parameters include the molecular weight, theoretical pI, amino acid composition, atomic composition, extinction coefficient, estimated half-life, instability index, aliphatic index, and grand average of hydropathicity (GRAVY). </p>
    </div>
    <div>
    <h5 class="parameter-heading">4. Crule</h5>
    <p class="parameter-description">ProtParam is a tool that allows the computation of various physical and chemical parameters for a given protein stored in Swiss-Prot or TrEMBL or for a user-entered protein sequence. The computed parameters include the molecular weight, theoretical pI, amino acid composition, atomic composition, extinction coefficient, estimated half-life, instability index, aliphatic index, and grand average of hydropathicity (GRAVY). </p>
    </div>
    <div>
    <h5  class="parameter-heading">5. ATrule</h5>
    <p class="parameter-description">ProtParam is a tool that allows the computation of various physical and chemical parameters for a given protein stored in Swiss-Prot or TrEMBL or for a user-entered protein sequence. The computed parameters include the molecular weight, theoretical pI, amino acid composition, atomic composition, extinction coefficient, estimated half-life, instability index, aliphatic index, and grand average of hydropathicity (GRAVY). </p>
    </div>
    <div>
    <h5  class="parameter-heading">6. Protein induced form</h5>
    <p class="parameter-description">ProtParam is a tool that allows the computation of various physical and chemical parameters for a given protein stored in Swiss-Prot or TrEMBL or for a user-entered protein sequence. The computed parameters include the molecular weight, theoretical pI, amino acid composition, atomic composition, extinction coefficient, estimated half-life, instability index, aliphatic index, and grand average of hydropathicity (GRAVY). </p>
    </div>
    <div>
    <h5  class="parameter-heading">7. Stabilizing energy ZDNA</h5>
    <p class="parameter-description">ProtParam is a tool that allows the computation of various physical and chemical parameters for a given protein stored in Swiss-Prot or TrEMBL or for a user-entered protein sequence. The computed parameters include the molecular weight, theoretical pI, amino acid composition, atomic composition, extinction coefficient, estimated half-life, instability index, aliphatic index, and grand average of hydropathicity (GRAVY). </p>
    </div>
    <div>
    <h5  class="parameter-heading">8. Bending Stiffness</h5>
    <p class="parameter-description">ProtParam is a tool that allows the computation of various physical and chemical parameters for a given protein stored in Swiss-Prot or TrEMBL or for a user-entered protein sequence. The computed parameters include the molecular weight, theoretical pI, amino acid composition, atomic composition, extinction coefficient, estimated half-life, instability index, aliphatic index, and grand average of hydropathicity (GRAVY). </p>
    </div>
    <div>
    <h5 class="parameter-heading">9. Duplex Stability Free energy</h5>
    <p class="parameter-description">ProtParam is a tool that allows the computation of various physical and chemical parameters for a given protein stored in Swiss-Prot or TrEMBL or for a user-entered protein sequence. The computed parameters include the molecular weight, theoretical pI, amino acid composition, atomic composition, extinction coefficient, estimated half-life, instability index, aliphatic index, and grand average of hydropathicity (GRAVY). </p>
    </div>
    <div>
    <h5  class="parameter-heading">10. DNA denaturation</h5>
    <p class="parameter-description">ProtParam is a tool that allows the computation of various physical and chemical parameters for a given protein stored in Swiss-Prot or TrEMBL or for a user-entered protein sequence. The computed parameters include the molecular weight, theoretical pI, amino acid composition, atomic composition, extinction coefficient, estimated half-life, instability index, aliphatic index, and grand average of hydropathicity (GRAVY). </p>
    </div>
    <div>
    <h5  class="parameter-heading">11. Propeller Twist</h5>
    <p class="parameter-description">ProtParam is a tool that allows the computation of various physical and chemical parameters for a given protein stored in Swiss-Prot or TrEMBL or for a user-entered protein sequence. The computed parameters include the molecular weight, theoretical pI, amino acid composition, atomic composition, extinction coefficient, estimated half-life, instability index, aliphatic index, and grand average of hydropathicity (GRAVY). </p>
    </div>
    <div>
    <h5 class="parameter-heading">12. Stacking Energy</h5>
    <p class="parameter-description">ProtParam is a tool that allows the computation of various physical and chemical parameters for a given protein stored in Swiss-Prot or TrEMBL or for a user-entered protein sequence. The computed parameters include the molecular weight, theoretical pI, amino acid composition, atomic composition, extinction coefficient, estimated half-life, instability index, aliphatic index, and grand average of hydropathicity (GRAVY). </p>
    </div>           
    </div>
 <h3 class="param">Trinucleotide</h3>
 <div class="parameters-container m-5">
 <div class="parameter">
     <h5  class="parameter-heading">1. Bendability</h5>
     <p class="parameter-description">ProtParam is a tool that allows the computation of various physical and chemical parameters for a given protein stored in Swiss-Prot or TrEMBL or for a user-entered protein sequence. The computed parameters include the molecular weight, theoretical pI, amino acid composition, atomic composition, extinction coefficient, estimated half-life, instability index, aliphatic index, and grand average of hydropathicity (GRAVY).</p>
     </div>
     <div>
     <h5  class="parameter-heading">2. Nucleosomal positioning</h5>
     <p class="parameter-description">ProtParam is a tool that allows the computation of various physical and chemical parameters for a given protein stored in Swiss-Prot or TrEMBL or for a user-entered protein sequence. The computed parameters include the molecular weight, theoretical pI, amino acid composition, atomic composition, extinction coefficient, estimated half-life, instability index, aliphatic index, and grand average of hydropathicity (GRAVY).</p>
     </div>
     <div>
     <h5 class="parameter-heading">3. B form to A form transition</h5>
     <p class="parameter-description">ProtParam is a tool that allows the computation of various physical and chemical parameters for a given protein stored in Swiss-Prot or TrEMBL or for a user-entered protein sequence. The computed parameters include the molecular weight, theoretical pI, amino acid composition, atomic composition, extinction coefficient, estimated half-life, instability index, aliphatic index, and grand average of hydropathicity (GRAVY).</p>
     </div>
</div>
    

{% endblock content %}